back/engines/commercial/report_builder/app/services/report_builder/queries/users_by_custom_field/base.rb

```diff
@@ -0,0 +1,28 @@
+module ReportBuilder
+  class Queries::UsersByCustomField::Base < Queries::Base
+    def run_query(start_at: nil, end_at: nil, project_id: nil, **_other_props)
```

Comment:
In the end, I didn't include `group` here because `PropsLive` doesn't have it and it would be weird to add it there. Maybe we should change it with some other type improvements. E.g., currently, all our graph responses have unique type values in the types like `type: 'users_by_gender'`, but in reality, the server always sends `type: 'report_builder_data_units'`.